components/omega/doc/design/PGrad.md

```diff
@@ -0,0 +1,196 @@
+This will allow for the use of a $p^\star$ coordinate, which functions similarly to the $z^\star$ in the Boussinesq MPAS-Ocean model.
+
+### 2.2 Requirement: Initial support for a simple centered pressure gradient
+
+For initial global cases without ice shelf cavities, the pressure and geopotential gradients will be computed with a simple centered difference approximation. In later versions of Omega, one ore more  high-order pressure gradients will be implemented and will replace the centered approach in production runs.
+However, the centered pressure gradient will remain an option for use in idealized testing.
+
+### 2.3 Requirement: Flexibility to support a high-order pressure gradient
+The centered pressure gradient will be insufficient for future versions of Omega that include ice shelf cavities and high resolution shelf breaks.
+The pressure gradient framework should be flexible enough to support a high-order pressure gradient in the future.
+
+### 2.4 Requirement: Flexibility to support tidal forcing and sea level change
+In later versions of Omega, the pressure gradient will need to be able to include tidal forcing in the geopotential term.
+These tidal forcings include both the tidal potential and the self attraction and loading terms.
+Additionally, other changes to the geoid
+
+### 2.5 Requirement: Pressure gradient for barotropic mode
+
+For split barotropic-baroclinic timestepping, the pressure gradient should provide the bottom pressure gradient tendency in the barotropic mode.
+
+### Desired:
+
+## 3 Algorithmic Formulation
+The non-Boussinesq momentum equation is
+$$ \frac{D \mathbf{u}_h}{D t } + f\boldsymbol{k}\times \mathbf{u}_h + \left(v\nabla_A p + \nabla_A \phi \right) = \boldsymbol{\mathcal{F}}. $$
+
+where $\mathbf{u}_h$ is the horizontal velocity, $f$ is the Coriolis parameter, $v = \frac{1}{\rho}$ is the specific volume, $\rho = \rho(T,S,p)$ is the density, $p$ is the hydrostatic pressure, $\phi$ is the geopotential, and $\boldsymbol{\mathcal{F}}$ are the dissipative terms.
+The operator $\nabla_A$ is the gradient along a constant surface, $A$, and the total derivative is
+
+$$ \frac{D \mathbf{u}_h}{D t } = \left( \frac{\partial}{\partial t} \right)_A  + \mathbf{u}_h\cdot \nabla_A + \omega\frac{\partial}{\partial A}, $$
+
+where $\omega$ is the cross coordinate flow.
+In the layered non-Boussinesq equations, the prognostic variable is the pressure thickness $h_k$, so that the geometric thickness (in meters) is a diagnostic variable defined as:
+
+$$ \Delta z_k = v_k h_k. $$
+
+The pressure at vertical cell interfaces is the found by summing the pressure thicknesses:
+
+$$ p_{K+1/2} = p_{surf} + g\sum_{k=1}^K h_k. $$
+
+The geopotential at vertical cell integrates is found by summing the pressure thicknesses multiplied by the specific volume:
+
+$$ \phi_{K+1/2} = g\left(z_b + \sum_{k=K}^{N} v_k h_k\right), $$
```

Comment:
I think a lower case `k` and primed summation index `k'` is more standard. Upper case `K` is often used for a maximum index. But more generally, the important diagnostic variable is mid-level pressure, so I think that should be written here. See my pressure equation in the [Discrete Equation section](https://portal.nersc.gov/project/e3sm/mpeterse/omega/design-doc-v1-eqns/_build/html/design/OmegaV1GoverningEqns.html#discrete-equations).